models/incident.js

```diff
@@ -48,6 +48,16 @@ schema.post('save', function() {
   schema.emit('incident:save', {_id: this._id.toString()});
 });
 
+schema.post('remove', function() {
+  Report.find({_incident:this._id.toString()}, function(err,reports){
```

Comment:
I would like us to adhere to this style guide: https://github.com/felixge/node-style-guide. So e.g. spaces inside curly braces and one space after colon. I'm not super picky but it would be great if you could take a look.